public/views/signin.html

```diff
@@ -1,71 +1,91 @@
-<div id="main-bod" ng-controller="IndexController">
+<div id="main-body" ng-controller="IndexController">
   <!-- Fixed Logo & Menu Bar Start -->
-  <div id="top-part">
-    <div class="wpb-top-bar">
-      <div class="mod-contain" id="navver">
-        <div class="row">
-          <!-- Logo -->
-          <div class="wpb-logo-container">
-            <div id="landing-logo"> 
-              <span id="first-word">Cards</span>
-              <span id="second-word"> for</span> 
-              <span id="third-word">Humanity</span> 
-            </div> 
-            <div class="col-md-9 wpb-menu-container">
-              <ul class="wpb-main-menu">
-                <li><a href="/">Home</a></li>
-              </ul>
-            </div>
-          </div>      
-        </div>
-      </div>
+  <nav class="navbar navbar-expand-sm navbar-light navbar-inverse bg-light fixed-top ">
+    <a class="navbar-brand" href="/" style="cursor:pointer">
+      <span id="cards">Cards</span> For Humanity</a>
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent"
```

Comment:
If a tag has many attribute, it can be written as
```html
<button
  class="navbar-toggler"
  type="button"
  data-toggle="collapse"
  data-target="#navbarSupportedContent"
  aria-controls="navbarSupportedContent"
  aria-expanded="false"
  aria-label="Toggle navigation"
>
```

The tab space in the above html corresponds to 2 spaces